.github/workflows/terraform-ci.yml

```diff
@@ -0,0 +1,216 @@
+        # version, because "goenv" can react to it automatically.
+        run: |
+          echo "Building with Go $(cat .go-version)"
+          echo "go-version=$(cat .go-version)" >> "$GITHUB_OUTPUT"
+  go-fmt-and-lint-acceptance:
+    runs-on: ubuntu-latest
+    needs:
+      - get-go-version
+    defaults:
+      run:
+        working-directory: ./test/acceptance
+    steps:
+      - name: Checkout
+        uses: actions/checkout@ac593985615ec2ede58e132d2e21d2b1cbd6127c # v3.3.0
+      - name: Setup Go
+        uses: actions/setup-go@6edd4406fa81c3da01a34fa6f6343087c207a568 # v3.5.0
+        with:
+          go-version: ${{ needs.get-go-version.outputs.go-version }}
+      - name: go fmt
+        run: |
+          files=$(go fmt ./...)
+          if [ -n "$files" ]; then
+            echo "The following file(s) do not conform to go fmt:"
+            echo "$files"
+            exit 1
+          fi
+      - name: Go CI lint
+        uses: golangci/golangci-lint-action@08e2f20817b15149a52b5b3ebe7de50aff2ba8c5 # v3.4.0
+        with:
+          args: |
+            --verbose
+          only-new-issues: false
+          skip-pkg-cache: true
+          skip-build-cache: true
+          working-directory: ./test/acceptance
+      - name: Lint Consul retry
+        run: |
+          go install github.com/hashicorp/lint-consul-retry@latest
+          lint-consul-retry
+  terraform-fmt:
+    runs-on: ubuntu-latest
+    needs:
+      - action-lint
+    steps:
+    - name: Checkout
+      uses: actions/checkout@ac593985615ec2ede58e132d2e21d2b1cbd6127c # v3.3.0
+    - name: Setup Terraform
+      uses: hashicorp/setup-terraform@v2
+      with:
+        terraform_version: 1.4.2
+    - name: Validate
+      run: terraform fmt -check -recursive .
+  acceptance:
+    name: ${{ matrix.name }}
+    runs-on: ubuntu-latest
+    env:
+      TEST_RESULTS: /tmp/test-results
+      GOTESTSUM_VERSION: 1.8.0
+    needs:
+      - get-go-version
+      - terraform-fmt
+      - go-fmt-and-lint-acceptance
+    defaults:
+      run:
+        working-directory: ./test/acceptance
+    strategy:
+      # We have a limit of 6 HCP Consul clusters.
+      # The following controls whether to enable HCP when testing release branches.
+      # HCP is always disabled for tests on PRs.
+      matrix:
+        name:
+          - acceptance-1.12-FARGATE-HCP
+          - acceptance-1.13-FARGATE
+          - acceptance-1.14-FARGATE-HCP
+          - acceptance-1.12-EC2
+          - acceptance-1.13-EC2-HCP
+          - acceptance-1.14-EC2
+        include:
+          - name: acceptance-1.12-FARGATE-HCP
+            consul-version: '1.12.7'
+            enable-hcp: true
+            launch-type: FARGATE
+
+          - name: acceptance-1.13-FARGATE
+            consul-version: '1.13.4'
+            enable-hcp: false
+            launch-type: FARGATE
+
+          - name: acceptance-1.14-FARGATE-HCP
+            consul-version: '1.14.2'
```

Comment:
Bump these to the latest available versions?

Reply:
:+1: ya, I had that thought too. Will do.